Terminal.Gui/Core/View.cs

```diff
@@ -1134,7 +1134,7 @@ public void Clear (Rect regionScreen)
 		/// <param name="rcol">Absolute column; screen-relative.</param>
 		/// <param name="rrow">Absolute row; screen-relative.</param>
 		/// <param name="clipped">Whether to clip the result of the ViewToScreen method, if set to <see langword="true"/>, the rcol, rrow values are clamped to the screen (terminal) dimensions (0..TerminalDim-1).</param>
-		internal void ViewToScreen (int col, int row, out int rcol, out int rrow, bool clipped = true)
+		public void ViewToScreen (int col, int row, out int rcol, out int rrow, bool clipped = true)
```

Comment:
ScreenToView is public already so hopefully its ok to make this also public (@BDisp thoughts?).

Its very handy for drawing stuff to screen directly and translating between coordinate systems (e.g. to draw interconnected borders).

Reply:
I agree.